packages/chat/components/chat/__stories__/chat.stories.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import '../chat';
+import { html } from 'lit';
+
+// More on how to set up stories at: https://storybook.js.org/docs/web-components/writing-stories/introduction
+export default {
+  title: 'Components/Chat/Chat',
+  tags: ['autodocs'],
+};
+
+export const Default = {
+  render: () => html` 
+    <c4ai-chat
+      model="llama-2"
+      userprompt="You are Watson, you will answer all my questions."
+      api-url="https://bam-api.res.ibm.com/v1/generate"
+      
+      temperature="0.0"
```

Comment:
Maybe we can utilize the storybook controls here for some of the properties? That way user can change/play with certain aspects of the chat right from storybook.

example with the extended-button: https://github.com/carbon-design-system/carbon-for-ai/blob/main/packages/extended-button/__stories__/extended-button.stories.js#L37

Theres different types you can have in the controls (Boolean, select - which you provide an array so it gives user dropdown options to choose from, label - text the user can input/edit, etc.)

Reply:
Great idea, I'll implement that later.